src/util/saveArticles.ts

```diff
@@ -424,7 +423,7 @@ export async function treatVideo(mw: MediaWiki, dump: Dump, srcCache: KVS<boolea
 
     /* Scrape subtitle */
     for (const track of Array.from(videoEl.querySelectorAll('track'))) {
-        subtitles.push(await treatSubtitle(track, webUrlHost, mw, articleId));
+        subtitles.push(await treatSubtitle(track, mw.webUrlHost, mw, articleId));
```

Comment:
I think now `treatSubtitle` would be advantageously changed to `treatSubtitle(track, mw, articleId)`?!